src/modals/collectible-modal/collectible-modal.tsx

```diff
@@ -151,13 +157,24 @@ export const CollectibleModal = () => {
               minted={timestamp}
               owned={collectible.balance}
               royalties={royalties}
-              style={styles.marginBottom}
             />
           )}
 
           {!isLoading && !isPropertiesSelected && isAttributesExist && (
-            <CollectibleAttributes attributes={attributes} style={styles.marginBottom} />
+            <CollectibleAttributes attributes={attributes} />
           )}
+
+          <View style={styles.burnContainer}>
+            <TouchableWithAnalytics
+              Component={TouchableOpacity}
+              onPress={burnCollectible}
+              style={styles.burnButton}
+              testID={CollectibleModalSelectors.burnButton}
+            >
+              <Text style={styles.burnButtonText}>Burn Nft</Text>
```

Comment:
as I see, on the design this text is uppercase

Reply:
typography that I use has 'uppercase' style

Reply:
sorry, miss it